components/PastEventsCarousel.jsx

```diff
@@ -0,0 +1,85 @@
+import Image from 'next/image';
+import { SwiperSlide } from 'swiper/react';
+import Tile from '../components/Tile';
+import TextSection from './TextSection';
+import Carousel from '../components/Carousel';
+import { PAST_EVENTS } from '../data/events';
+import styles from '../components/RolesCarousel.module.css';
+import Heading from './Heading';
+import { UnderlineTypes } from '../utils/underlineType';
+import Link from 'next/link';
+import 'swiper/css/navigation';
+import PastEventImage from './PastEventImage';
+
+const CAROUSEL_CONTAINER = styles.swiperContainer;
+const ARROW_INIT = { left: 'swiper-button-prev', right: 'swiper-button-next' };
+const LEFT_NAVIGATION_ARROW = styles.swiperButtonPrev;
+const RIGHT_NAVIGATION_ARROW = styles.swiperButtonNext;
+
+const TILE_CONTAINER = 'flex flex-col xl:h-[650px] lg:h-[820px] md:flex-column md:h-[820px] h-[905px] rounded-[25px] p-[1.875rem] h-full';
```

Comment:
Here is why vercel is deploying a conflicting view, over here you specified 

`const TILE_CONTAINER = 'flex flex-col xl:h-[650px] lg:h-[820px] md:flex-column md:h-[820px] h-[905px] rounded-[25px] p-[1.875rem] h-full';`

Because you are setting height to full `h-full` and ALSO to 905px `h-[905px]`, its somewhat like a 50-50 of what will be displayed.  Try removing `h-full` and this should resolve the preview issues!

Reply:
Also, I just noticed that a bit of the 3rd slide is being cut off, try changing the height to something like `h-[930px]`, smth like that :)